Maps/ArchipelagoCampaign/HotS/ap_domination.SC2Map/MapScript.galaxy

```diff
@@ -5101,6 +5110,11 @@ bool gt_VictoryZagarasBaseDestroyed_Func (bool testConds, bool runActions) {
     }
 
     TriggerEnable(TriggerGetCurrent(), false);
+    if ((gv_eggsCollected < 100)) {
+        lib5BD4895D_gf_AP_Core_collectLocationUpdateObjective(gv_LOCATION_ID_WIN_WITHOUT_100_EGGS);
+        TriggerExecute(gt_ObjectiveWithoutEggsComplete, true, false);

```

Comment:
If you win without collecting 100 eggs, you should probably also get the 100 eggs location as well on victory (to avoid forcing players having to play a mission again that they've already beaten.)